docs/ru/ddrone.md

```diff
@@ -0,0 +1,33 @@
+# Граффити коптер D-drone
+
+<iframe width="560" height="315" src="https://youtu.be/ErtioCj5iMw" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
This iframe didn't show up for me, for some reason:

<img width="603" alt="Screenshot 2021-03-12 at 10 25 09" src="https://user-images.githubusercontent.com/1683279/110906624-3ddb3b00-831d-11eb-93af-ca7e48d23901.png">

Are you sure the code is correct?